code/tony/python/lab02_version1.py

```diff
@@ -0,0 +1,3 @@
+nums = list()
```

Comment:
This is an empty list. It is also an interesting way to insatiate a list. Instead you can just say `nums = []`